client/extensions/woocommerce/app/products/product-update.js

```diff
@@ -141,20 +141,22 @@ class ProductUpdate extends React.Component {
 	};
 
 	onSave = () => {
-		const { product, translate } = this.props;
-
-		const successAction = successNotice(
-			translate( '%(product)s successfully updated.', {
-				args: { product: product.name },
-			} ),
-			{
-				duration: 8000,
-				button: translate( 'View' ),
-				onClick: () => {
-					window.open( product.permalink );
-				},
-			}
-		);
+		const { product, translate, site, fetchProductCategories: fetch } = this.props;
+		const successAction = () => {
+			fetch( site.ID );
+			return successNotice(
+				translate( '%(product)s successfully updated.', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **90** times:
`translate( '%1$s updated successfully.' )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/store-product-cat-query)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).